libs/react/betterangels-admin/src/lib/providers/user/UserProvider.graphql

```diff
@@ -1,4 +1,4 @@
-query currentUser {
+query currentOrgUser {
```

Comment:
@tglaz I considered updating `TUser` to `TOrgUser` but didn't want to do it without more input. Let me know if you think it's a good idea